### PR TITLE
Script to insert odour category to fixmystreet.com

### PR DESCRIPTION
**Pull request by MorayMySoc**
Add Environment Agency Odor category to FixMyStreet.com #2110

* Environment Agency would like odour issues in England to be emailed to them
* Model script on update_areas_covered
* Create Environment Agency body
* Add Odour category with dropdown and notes
* Add areas this applies to, to the new body

### CODE REVIEW

**Review by struan — CHANGES_REQUESTED** [review]
I've had a quick check and we're also going to need to stop this category appearing on cobrands. 

The categories to display are set up in https://github.com/mysociety/fixmystreet/blob/ae7bc51ef24f5960c71be65af391978ca23185b3/perllib/FixMyStreet/App/Controller/Report/New.pm#L732 so have a look at that and see if it makes sense.

File: bin/fixmystreet.com/one-off-add-ics-body [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		{ key => '1', name => 'Very faint odour' },
+		{ key => '2', name => 'Faint odour' },
+		{ key => '3', name => 'Distinct odour' },
+		{ key => '4', name => 'Strong odour' },
+		{ key => '5', name => 'Very strong odour' },
+		{ key => '6', name => 'Extremely strong odour' },
+	],
+},
+);
+$cat->insert;
```
[the remark this answers is not in the document]
Comment (struan) [reply bin/fixmystreet.com/one-off-add-ics-body:86]: Ah right, some of this is tabs and some is spaces is I think the problem. it should be spaces.

**Review by struan — APPROVED** [review]
indenting still a bit off in the script but this looks good.

**Comment by MorayMySoc** [comment]
Merged and released

**Pull request closed without merging**